### PR TITLE
Fix theme for messagebox

### PR DESCRIPTION
**Pull request by sleepwalky**
# What does this PR do:

- Add className prop

- Fix success color css-var

# Where should the reviewer start:

diff

### CODE REVIEW

File: components/messageBox/messageBox.js [hunk 1]
```diff
@@ -1,9 +1,19 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import classnames from 'classnames';
 import { getClassNamesWithMods, getDataAttributes } from '../_helpers';
 
-const MessageBox = ({ children, dataAttrs, icon, mods, title, type }) => {
-  const className = getClassNamesWithMods('ui-messageBox', [...mods, type]);
+const MessageBox = (props) => {
+  const {
+    children,
```
Comment (juliamaksimchik) [inline components/messageBox/messageBox.js]: Pls, sort it

Comment (sleepwalky) [reply components/messageBox/messageBox.js]: But... isn't it sorted?

File: themes/_default.yaml [hunk 2]
```diff
@@ -400,7 +400,7 @@ messagebox:
   background-color:
     info: "var(--tx-generic-color-secondary-light)"
     error: "var(--tx-generic-color-negative-light)"
-    success: "var(--tx-generic-color-positive-lighter)"
+    success: "var(--tx-generic-color-positive-light)"
```
Comment (iwwwi) [inline themes/_default.yaml]: is this according to our designs?

Comment (sleepwalky) [reply themes/_default.yaml]: @iwwwi 
well, I've just fixed typo

Comment (maximuk) [reply themes/_default.yaml]: We have this color in `travix-themes` and you shouldn't change the variable. I think you can just set any color for `positive-lighter` in `_default.yaml` file

Comment (sleepwalky) [reply themes/_default.yaml]: @iwwwi @maximuk 
I've rolled it back, guys!

**Pull request merged**